genesis/bin/genesis_playback.rs

```diff
@@ -297,13 +310,20 @@ fn main() {
     println!("Done, genesis state root is {:?}.", state_root);
 
     // Generate genesis roothash block file.
-    let mut block = roothash::Block::default();
-    block.header.namespace = runtime_id;
-    block.header.state_root = state_root;
-    block.header.io_root = Hash::empty_hash();
-    let blocks = vec![block];
+    let rtg = registry::RuntimeGenesis {
+        state_root: state_root,
+        state: vec![],
+        storage_receipts: vec![SignatureBundle {
+            // public_key must not be None, but empty.
+            public_key: Some(PublicKey::default()),
+            signature: Signature::default(),
+        }],
+        round: 0,
+    };
+    let rtg_map: HashMap<roothash::Namespace, registry::RuntimeGenesis> =
+        [(runtime_id, rtg)].iter().cloned().collect();
 
     // Save to file.
     let mut file = File::create(matches.value_of("output_file").unwrap()).unwrap();
-    serde_json::to_writer(&mut file, &blocks).unwrap();
+    serde_json::to_writer(&mut file, &rtg_map).unwrap();
```

Comment:
Actually, are you sure this is correct? As far as I see the CLI tooling expects this to be wrapped in another dictionary under `runtime_states`.

Reply:
Oh nevermind.